src/init.cpp

```diff
@@ -1389,16 +1389,19 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     assert(!node.connman);
     node.connman = MakeUnique<CConnman>(GetRand(std::numeric_limits<uint64_t>::max()), GetRand(std::numeric_limits<uint64_t>::max()), args.GetBoolArg("-networkactive", true));
 
+    // Sanity checks off by default for performance, because otherwise
```

Comment:
Good idea to keep this comment. Perhaps move it to the comment for `nCheckFrequency` in txmempool.h